aws/adapter.go

```diff
@@ -205,15 +266,38 @@ func (a *Adapter) FindManagedStacks() ([]*Stack, error) {
 	if err != nil {
 		return nil, err
 	}
+	return stacks, nil
+}
+
+// Update Auto Scaling Groups to have relevant Target Groups. Also
```

Comment:
exported functions should have godoc way of documentation, so you should start with "UpdateTargetGroupsAndAutoScalingGroups ..".